ci-operator/step-registry/ipi/conf/fips-sshkey/ipi-conf-fips-sshkey-commands.sh

```
@@ -0,0 +1,54 @@
+        echo "ERROR: ed25519 keys are not FIPS-compatible and must not be used"
+        exit 1
+    fi
+    key_file="/tmp/key-${key_type}"
+    option=""
+    if [[ "${key_type}" == "ecdsa" ]]; then
+        option="-b 521"
+    elif [[ "${key_type}" == "rsa" ]]; then
+        option="-b 4096"
+    fi
+    echo "Generating FIPS-compatible ssh key with type ${key_type}..."
+    ssh-keygen -t "${key_type}" ${option} -N '' -f "${key_file}"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)